src/Doctrine/Schema/DcaSchemaProvider.php

```diff
@@ -387,8 +408,8 @@ private function getSqlDefinitions(): array
     private function getMaximumIndexLength(Table $table, string $column): int
     {
         $indexLength = $this->getDefaultIndexLength($table);
+        $collation = $table->getOption('collate');
```

Comment:
Are you sure this is correct? The documentation says this should be `collation` (http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/schema-representation.html#vendor-specific-options). Not sure if that's for fields only though.

Reply:
Yes, I am sure. The default table options are merged:

<img width="559" alt="" src="https://user-images.githubusercontent.com/1192057/34821022-a03523be-f6c2-11e7-9116-799edb6153ff.png">

Reply:
I'd stumbled across this as well - ``collation`` for fields ``colate`` for tables, extremely unintuitive... :-/